src/main/java/com/incognia/SignupResponse.java

```diff
@@ -0,0 +1,13 @@
+package com.incognia;
+
+import java.util.Map;
+import java.util.UUID;
+import lombok.Value;
+
+@Value
+public class SignupResponse {
+  UUID id;
+  UUID requestId;
+  Assessment riskAssessment;
+  Map<String, Object> evidence;
```

Comment:
decided to keep this as a map at first, for simplicity. we can add getters for some evidences if we need it